docs/source/user_guides/basic_conceptions.rst

```diff
@@ -139,165 +148,185 @@ That's what the agent consists of:
+    in the `Context` object under unique ids and abstracting the storage logic away from the user interface.
+    As a result, it offers the basic level of data protection making it impossible to gain unlawful access to personal information.
 
-* You can serialize context (available on every transition and response)
-  to json or dictionary in order to debug it or extract some values.
-  See `tutorial on context serialization`_.
+Documentation
+~~~~~~~~~~~~~
 
-* You can alter user input and modify generated responses.
-  User input can be altered with ``PRE_RESPONSE_PROCESSING`` and will happen **before** response generation.
-  See `tutorial on pre-response processing`_.
-  Node response can be modified with ``PRE_TRANSITION_PROCESSING`` and will happen **after** response generation.
-  See `tutorial on pre-transition processing`_.
+Creating documentation is essential for teamwork and future bot maintenance. 
+Document how the intent works, its parameters, and expected outcomes. 
+This documentation serves as a reference for developers and stakeholders involved in the project.
 
-* Additional data ``MISC`` can be added to every node, flow and script itself.
-  See `tutorial on script MISC`_.
+Scaling
+~~~~~~~
 
-Conclusion
-~~~~~~~~~~
+If your bot becomes popular and requires scaling, consider scalability during development.
+Scalability ensures that the bot can handle a growing user base without performance issues.
+While having only one application instance will suffice in most cases, there are many ways
+how you can adapt the application to a high load environment.
 
-In this tutorial, we explored the basics of Dialog Flow Framework (DFF) to build dynamic conversational services.
-By using DFF's intuitive Domain-Specific Language (DSL) and well-structured dialog graphs, we created a simple interaction between user and chat-bot.
-We covered installation, understanding the DSL and building dialog graph.
-However, this is just the beginning. DFF offers a world of possibilities in conversational chat-bot.
-With practice and exploration of advanced features, you can create human-like conversations and reach a wider audience by integrating with various platforms.
-Now, go forth, unleash your creativity, and create captivating conversational services with DFF.
-Happy building!
+* With the database connection support that DFF offers out of the box, DFF projects can be easily scaled through sharing the same database between multiple application instances. However, using an external database is required due to the fact that this is the only kind of storage that can be efficiently shared between processes.
+* Likewise, using multiple database instances to ensure the availability of data is also an option.
+* The structure of the `Context` object makes it easy to shard the data storing different subsets of data across multiple database instances.
 
+Further reading
```

Comment:
This section is empty.
![image](https://github.com/deeppavlov/dialog_flow_framework/assets/61429541/bfa05865-f7f9-4cd3-aa5a-78cb88f7f666)